src/rgw/rgw_admin.cc

```diff
@@ -309,6 +309,7 @@ void usage()
   cout << "   --display-name=<name>     user's display name\n";
   cout << "   --max-buckets             max number of buckets for a user\n";
   cout << "   --admin                   set the admin flag on the user\n";
+  cout << "   --reader                  set the reader characteristics on the user\n";
```

Comment:
* i worry that the term `reader` is way too vague here
* `--reader` appears to just set the `--admin` and `--op-mask` options; if we already have full control over these options, why bother with this new flag?

Reply:
The main reason why I didn't want to use --op-mask is that I see no documentation whatsoever as to what the bits in the mask are.

Note that this whole section is not required to support "S-RBAC" as OpenStack defines it, and in fact it does something else: it sets op_mask on whole identities (accounts), not at roles. I only kept it for some quick testing.

I think the easiest is to drop all of that.